msteams-platform/resources/moodleInstructions.md

```diff
@@ -83,157 +83,169 @@ Next, you'll need to register Moodle as an application in your Azure AD. We've p
     * Right-click Windows PowerShell.
-    * Select "Run as Administrator".
+    * Select **Run as Administrator**.
 
 1. Navigate to the unzipped directory by typing `cd .../.../Moodle-AzureAD-Powershell` where `.../...` is the path to the directory.
 
 1. Execute the PowerShell script:
 
     * Enter `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`.
     * Enter `./Moodle-AzureAD-Script.ps1`.
-    * Login to your Microsoft 365 administrator account in the pop-up window.
+    * Sign in to your Microsoft 365 administrator account in the pop-up window.
     * Enter the name of the Azure AD Application (e.g., Moodle/Moodle plugin).
     * Enter the URL for your Moodle server.
     * Copy the **Application ID** and **Application Key** generated by the script and save them.
 
-1. Next you'll need to add the` AppID` and `Key` to the Microsoft 365 Moodle Plugin. Return to the plugin administration page (Site administration ➡ Plugins ➡ Microsoft 365 Integration).
+1. Next you must add the `AppID` and `Key` to the Microsoft 365 Moodle Plugin. Return to the plugin administration page. The flow is: Site administration ➡ Plugins ➡ Microsoft 365 Integration.
 
 1. On the **Setup** tab add the **Application ID** and **Application Key** you copied previously, then select **Save changes**.
 
-1. Once the page refreshes you should see a new section **Choose connection method**. Select the checkbox labeled **Default** and then select **Save changes** again.
+1. Once the page refreshes, you can see a new section **Choose connection method**. Select the checkbox labeled **Default** and then select **Save changes** again.
 
-1. Once the page refreshes you will see another new section **Admin consent & additional information**.
-    * Select the **Provide Admin Consent** link, enter your Microsoft 365 Global Administrator credentials, then **Accept** to grant the permissions.
-    * Next to the **Azure AD Tenant** field select the **Detect** button.
-    * Next to the **OneDrive for Business URL** select the **Detect** button.
+1. Once the page refreshes you can see another new section **Admin consent & additional information**.
+    * Select **Provide Admin Consent** link, enter your Microsoft 365 Global Administrator credentials, then **Accept** to grant the permissions.
+    * Next to the **Azure AD Tenant** field, select the **Detect** button.
+    * Next to the **OneDrive for Business URL**, select the **Detect** button.
     * Once the fields populate, select the **Save changes** button again.
 
 1. Select the **Update** button to verify the installation, then **Save changes**.
 
-1. Next, you'll need to synchronize users between your Moodle server and Azure AD. Depending on your environment, you may select different options during this stage. To get started:
+1. Next, you must synchronize users between your Moodle server and Azure AD. Depending on your environment, you can select different options during this stage. To get started:
```

Comment:
Do not start the step with "Next, you must...". Step should directly tell the user what needs to be done. For example, here it should be:
1. Synchronize users between your Moodle server and Azure AD.